README.md

```diff
@@ -1,8 +1,10 @@
 # Optimizely VSCode Extension
 
-The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript typescript and react.  If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
+The Optimizely VSCode extension lets you access Feature Keys, Experiment Keys and Feature Variables via auto-complete for javascript, typescript and react. If not using these languages, you can use the keyboard shortcuts below to insert your experiment and feature keys as well as attribute and event keys.
 
-This extension is compatiable with 1.34 and higher of vscode.
+This extension also evaluates the use of Optimizely's public APIs (e.g. getFeatureVariableJSON(feature_key, userId, attributes)) in active .js and .ts files, and highlights Feature Keys that do not exist in datafile.
```

Comment:
I'd reword as:
This extension also evaluates if parameter values are valid for Optimizely's public SDK methods (for example, getFeatureVariable(feature_key, userID, attributes), or isFeatureEnabled(feature_key)) for Javascript and Typescript. If a given parameter is invalid (i.e., doesn't exist in the datafile), this extension highlights it and displays a warning on hover.

Reply:
I think this is great! I would just specify what parameter values being evaluated, which are only feature keys at the moment:

This extension also evaluates if `feature keys` are valid for Optimizely's public SDK methods (for example, getFeatureVariable(feature_key, userID, attributes), or isFeatureEnabled(feature_key)) for Javascript and Typescript. If a given `feature_key` is invalid (i.e., doesn't exist in the datafile), this extension highlights it and displays a warning on hover.